### DISCUSSION TITLE
The download for osx-multi

### DISCUSSION THREAD

**Discussion by wasp-codes** [philippe44/AirConnect#428] [General]
Thanks for the great app! I've been using it for the past year and have had no problems. Just wanted to redownload after a clean install on my computer and I've noticed that the file for intel macs, which is the osx-multi version is missing from the bin folder. Has this been discontinued?

Thanks again!

**Comment by philippe44** [comment]
No, it's under v2 branch. I'm currently doing refactoring and osx, solaris and freebsd builds are gone for now
